furbaby/api/views.py

```diff
@@ -177,3 +187,125 @@ def password_reset_token_created(sender, instance, reset_password_token, *args,
     )
     msg.attach_alternative(email_html_message, "text/html")
     msg.send()
+
+
+@api_view(["GET", "POST", "OPTIONS", "PUT", "PATCH", "DELETE"])
+def user_location_view(request):
+    location_view = UserLocationView()
```

Comment:
The class is defined after the usage, I'd suggest you move the class to a different file or move it above it's usage.